lib/sidekiq-scheduler/manager.rb

```diff
@@ -18,21 +18,28 @@ def initialize(options)
       Sidekiq::Scheduler.enabled = options[:enabled]
       Sidekiq::Scheduler.dynamic = options[:dynamic]
       Sidekiq::Scheduler.listened_queues_only = options[:listened_queues_only]
-      Sidekiq.schedule = options[:schedule]
+
+      @schedule = options[:schedule]
+      @enabled = options[:enabled]
     end
 
     def stop
       Sidekiq::Scheduler.clear_schedule!
     end
 
     def start
-      Sidekiq::Scheduler.load_schedule!
+      if enabled
+        Sidekiq.schedule = schedule
+        Sidekiq::Scheduler.load_schedule!
```

Comment:
By making this change to `#initialize`, the change in `#start` will not be needed